testing/src/test/java/io/stargate/it/http/RestApiTest.java

```diff
@@ -314,6 +314,54 @@ public void getRowBadRequest() throws IOException {
         HttpStatus.SC_BAD_REQUEST);
   }
 
+  @Test
+  public void getRowWithClustering() throws IOException {
+    String tableName = "tbl_getallrows_clustering_" + System.currentTimeMillis();
+    createTableWithClustering(tableName);
+
+    String id = UUID.randomUUID().toString();
+    List<ColumnModel> columns = new ArrayList<>();
+    columns.add(new ColumnModel("id", id));
+    columns.add(new ColumnModel("expense_id", "1"));
+    columns.add(new ColumnModel("firstName", "John"));
+    addRow(tableName, columns);
+
+    columns = new ArrayList<>();
+    columns.add(new ColumnModel("id", id));
+    columns.add(new ColumnModel("expense_id", "2"));
+    columns.add(new ColumnModel("firstName", "Jane"));
+    addRow(tableName, columns);
+
+    columns = new ArrayList<>();
+    columns.add(new ColumnModel("id", UUID.randomUUID().toString()));
+    columns.add(new ColumnModel("expense_id", "3"));
+    columns.add(new ColumnModel("firstName", "Alice"));
+    addRow(tableName, columns);
+
+    String body =
```

Comment:
Looking at this so java-ish code...
I wonder if instead we could use sth more like http://rest-assured.io/
so that test cases will become self-explaining?

Reply:
Interesting, we'll have to look into that. cc @EricBorczuk